tf_encrypted/keras/layers/layers_utils.py

```diff
@@ -0,0 +1,13 @@
+# pylint: disable=arguments-differ
+"""Utilities for layers"""
+
+
+def normalize_data_format(value):
```

Comment:
I see it's used in convolutions as well! Still think renaming is relevant.

Reply:
I think we should move this function to conv_utils.py, since that's where it lives in normal keras and it's only really useful for conv-related layers

Reply:
after thinking on this more, I think we should be using the installed tensorflow.keras.backend's value for data_format, so we can remove this whole function and just use that one

Reply:
yeah, I think that should work.